src/scss/_config.scss

```diff
@@ -0,0 +1,72 @@
+/**
+ * IMPORTED TOKENS
+ * These are generated with the design tokens data
+*/
+@import 'tokens';
+
+/**
+ * IMPORTED THEMES
+ * These themes are for dark/light mode and abstract CSS Custom
+ * property references to design tokens into more contextual references
+*/
+@import 'themes';
+
+/**
+ * CORE CONFIG
+ * This powers everything from utility class generation to breakpoints
+ * to enabling/disabling pre-built components/utilities.
+ */
+$gorko-config: (
+  'css-vars': (
+    'color': $gorko-colors,
+    'themes': $gorko-themes
+  ),
+  'bg': (
+    'items': $gorko-theme-keys,
+    'output': 'standard',
+    'property': 'background'
+  ),
+  'color': (
+    'items': $gorko-theme-keys,
+    'output': 'standard',
+    'property': 'color'
+  ),
+  'flow-space': (
+    'items': $gorko-space-scale,
+    'output': 'standard',
+    'property': '--flow-space'
+  ),
+  'font': (
+    'items': $gorko-fonts,
+    'output': 'standard',
+    'property': 'font-family'
+  ),
+  'gap-top': (
+    'items': $gorko-space-scale,
+    'output': 'standard',
+    'property': 'margin-top'
+  ),
+  'leading': (
+    'items': (
+      'flat': '1.1'
+    ),
+    'output': 'standard',
+    'property': 'line-height'
+  ),
+  'measure': (
+    'items': (
+      'short': '40ch',
+      'long': '65ch'
+    ),
+    'output': 'standard',
+    'property': 'max-width'
+  ),
+  'text': (
+    'items': $gorko-size-scale,
+    'output': 'standard',
+    'property': 'font-size'
+  ),
+  'breakpoints': (
+    'md': '(min-width: 37em)'
```

Comment:
Maybe worth mentioning... on d.c.c. we originally were using min-width, em based breakpoints. But we changed to using pixel based ones because we occasionally ran into issues where we needed a max-width breakpoint for whatever reason. So we ended up with something like this (https://github.com/GoogleChrome/developer.chrome.com/blob/9e3e5f2bc0dd38ba2f0d27ec13302f0116c0569b/site/_scss/globals/gorko/_config.scss#L342-L349) which is a bit gross but was sort of built up over time based on actual needs.

I'm not sure how best to handle the occasional max-width breakpoint using ems?

Reply:
Yeh I don't mind using pixels at all—especially if you've had problems already. I wasn't planning on doing breakpoints until I started doing some global CSS, so let's park this particular bit.